EnviroSense.Domain/Emailing/SendPasswordChangedNotificationEmailPayload.cs

```diff
@@ -0,0 +1,9 @@
+﻿using System.Runtime.InteropServices.JavaScript;
+
+namespace EnviroSense.Domain.Emailing;
+
+public class SendPasswordChangedNotificationEmailPayload : BaseEmail
+{
+    public required DateTime LoginDate { get; set; }
```

Comment:
```suggestion
    public required DateTime ChangeDate { get; set; }
```

will probably have better meaning